src/raft_baselines/scripts/raft_predict.py

```diff
@@ -40,7 +40,7 @@ def base_config():
     classifier_name = "GPT3Classifier"
     classifier_kwargs = {
         # change to davinci to replicate results from the paper
-        "engine": "ada",
+        # "engine": "ada",
```

Comment:
I had to comment this out because it's not an attribute of the `transformers` classifiers